src/utils/link/link-by-prompt.js

```diff
@@ -199,7 +200,7 @@ Run ${chalk.cyanBright('`git remote -v`')} to see a list of your git remotes.`))
   context.log(`Site url:  ${chalk.cyanBright(site.ssl_url || site.url)}`)
   context.log()
 
-  context.log(`Site id saved to ${context.netlify.site.root}/.netlify/state.json`)
+  context.log(`Site id saved to ${path.join(context.netlify.site.root, '/.netlify/state.json')}`)
```

Comment:
does `path.join` replace `/` with `\` in windows?

I'd use this to ensure the separator is consistent:

```
path.join(context.netlify.site.root, '.netlify', 'state.json')
```

Reply:
Of course it does, just check the docs. Nothing else is needed ;)

Reply:
Agreed @calavera